parastell/nwl_utils.py

```diff
@@ -145,7 +145,7 @@ def find_coords(data):
             bounds=[(-np.pi, np.pi)],
             args=(vmec, wall_s, coords[0], coords[1]),
         )
-        thetas.append(theta.x[0])
+        thetas.append((theta.x[0] + 2 * np.pi) % (2 * np.pi))
```

Comment:
For our future benefit, it's best to add a comment here explaining (briefly) this conversion